src/main/router/dir-router.ts

```diff
@@ -3,6 +3,19 @@ import { none, some } from "../lib/rust-like-utils-backend/Optional";
 import { dialog } from "electron";
 import path from "path";
 
+Router.respond("dir::openFile", async (_evt, options) => {
+  const result = await dialog.showOpenDialog({
+    properties: ["openFile"],
+    ...options,
+  });
+
+  if (result.canceled || result.filePaths.length === 0) {
+    return none();
+  }
+
+  return some(result.filePaths[0]);
```

Comment:
Is using the path of the image ok? Wouldn't it be better to copy the image in the application storage and use that path instead?

Reply:
Copying the image would take more space on the user device unnecessarily.

Reply:
Right, i was worried that when deleting the image locally it wouldn't load anymore but i guess it's not a big problem

Reply:
This is the down side of this approach, we lose the reference to the image if it is deleted or moved, but the fact that we don't have to copy the image far outweighs this.